lib/ansible/modules/remote_management/manageiq/manageiq_group.py

```diff
@@ -0,0 +1,596 @@
+    type: list
+    required: false
+    default: null
+  belongsto_filters_merge_mode:
+    description:
+    - In merge mode existing settings are merged with the supplied C(belongsto_filters).
+    - In replace mode current values are replaced with the supplied C(belongsto_filters).
+    choices: [ merge, replace ]
+    default: replace
+'''
+
+EXAMPLES = '''
+- name: Create a group in ManageIQ with the role EvmRole-user and tenant 'my_tenant'
+  manageiq_group:
+    description: 'MyGroup-user'
+    role: 'EvmRole-user'
+    tenant: 'my_tenant'
+    manageiq_connection:
+      url: 'https://manageiq_server'
+      username: 'admin'
+      password: 'smartvm'
+      verify_ssl: False
+
+- name: Create a group in ManageIQ with the role EvmRole-user and tenant with tenant_id 4
+  manageiq_group:
+    description: 'MyGroup-user'
+    role: 'EvmRole-user'
+    tenant_id: 4
+    manageiq_connection:
+      url: 'https://manageiq_server'
+      username: 'admin'
+      password: 'smartvm'
+      verify_ssl: False
+
+- name: Create or update a group in ManageIQ with the role EvmRole-user and tenant my_tenant, prov_max_cpu tags, limit access to a cluster
+  manageiq_group:
+    description: 'MyGroup-user'
+    role: 'EvmRole-user'
+    tenant: my_tenant
+    managed_filters:
+    - - "/managed/prov_max_cpu/1"
+      - "/managed/prov_max_cpu/3"
+      - "/managed/prov_max_cpu/4"
+    managed_filters_merge_mode: replace
+    belongsto_filters:
+    - "/belongsto/ExtManagementSystem|ProviderName/EmsFolder|Datacenters/EmsFolder|dc_name/EmsFolder|host/EmsCluster|Cluster name"
+    belongsto_filters_merge_mode: merge
+    manageiq_connection:
+      url: 'https://manageiq_server'
+      username: 'admin'
+      password: 'smartvm'
+      verify_ssl: False
+
+- name: Delete a group in ManageIQ
+  manageiq_group:
+    state: 'absent'
+    description: 'MyGroup-user'
+    manageiq_connection:
+      url: 'http://127.0.0.1:3000'
+      username: 'admin'
+      password: 'smartvm'
+
+- name: Delete a group in ManageIQ using a token
+  manageiq_group:
+    state: 'absent'
+    description: 'MyGroup-user'
+    manageiq_connection:
+      url: 'http://127.0.0.1:3000'
+      token: 'sometoken'
+'''
+
+RETURN = '''
+group:
+  description: The group.
+  returned: success
+  type: complex
+  contains:
+    description:
+      description: The group description
+      returned: success
+      type: string
+    id:
+      description: The group id
+      returned: success
+      type: int
+    group_type:
+      description: The group type, system or user
+      returned: success
+      type: string
+    role_name:
+      description: The group role name
+      returned: success
+      type: string
+    tenant_name:
+      description: The group tenant name
+      returned: success
+      type: string
+    created_on:
+      description: Group creation date
+      returned: success
+      type: int
```

Comment:
`2018-08-12T08:37:55+00:00` is not an int :)

Reply:
changed to string